kong/clustering/data_plane.lua

```diff
@@ -80,10 +80,41 @@ function _M:init_worker(basic_info)
   self.plugins_list = basic_info.plugins
   self.filters = basic_info.filters
 
-  -- only run in process which worker_id() == 0
-  assert(ngx.timer.at(0, function(premature)
-    self:communicate(premature)
-  end))
+  local function start_communicate()
+    assert(ngx.timer.at(0, function(premature)
+      self:communicate(premature)
+    end))
+  end
+
+  -- does not config rpc sync
+  if not kong.sync then
```

Comment:
sync is rpc's service, so should we check this `if not kong.rpc then`?

Reply:
No, currently the only job of clustring/data_plane is syncing, it should work if sync v2 is disabled.